scalding-core/src/main/scala/com/twitter/scalding/FutureCache.scala

```diff
@@ -2,45 +2,73 @@ package com.twitter.scalding
 
 import java.util.concurrent.ConcurrentHashMap
 import scala.concurrent.{ Future, Promise, ExecutionContext => ConcurrentExecutionContext }
+
+/**
+ * Represents a cancellable future.
+ */
+case class CFuture[+T](future: Future[T], cancellationHandler: CancellationHandler)
```

Comment:
can we move this to a separate file to make it easier to find for non-IDE users?

Reply:
👍